plugins/jdg70/src/main/resources/server.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" ?>
+
+<server xmlns="urn:jboss:domain:4.0">
```

Comment:
To be clear, I don't require the above in this PR, just my eyes hurt from hundreds of lines just copied, and I've hijacked your PR for discussion.

So there are two ways to reduce the configuration:
a) use only one configuration file (if that's possible) and symlink other versions of plugins to that (so if you just copied this, you would use symlink instead).
b) use default server configs; as we need few modifications, keep a list of them in the conf/ dir and apply them (do the XML transformation) in RG.